modules/rostests/apitests/acpi/Bus_PDO_QueryResourceRequirements.c

```diff
@@ -501,7 +501,7 @@ START_TEST(Bus_PDO_QueryResourceRequirements)
     ok(ReqList->InterfaceType == Internal, "InterfaceType = %u\n", ReqList->InterfaceType);
     ok(ReqList->BusNumber == 0, "BusNumber = %lu\n", ReqList->BusNumber);
     ok(ReqList->SlotNumber == 0, "SlotNumber = %lu\n", ReqList->SlotNumber);
-    ok(ReqList->AlternativeLists == 2, "AlternativeLists = %lu\n", ReqList->AlternativeLists);
+    ok(ReqList->AlternativeLists == 1, "AlternativeLists = %lu\n", ReqList->AlternativeLists);
```

Comment:
This is not right. This test is a true unit test, it runs against ROS's implementation of this function even on Windows.
So we could either find a way to skip it on Windows, or mark these two failing tests as todo.

Reply:
Removed that part